statler_api/admin.py

```diff
@@ -1,11 +1,25 @@
 from django.contrib import admin
-from .models import PlayDAO
-from .models import ReviewDAO
-from .models import PlayListDAO
-from .models import PlayListEntryDAO
+from django import forms
+from .models import *
+
+# these 2 classes make it possible to add plays to a play list
+# without manually making a PlayListEntryDAO
+class PlayListEntryInline(admin.TabularInline):
+    model = PlayListEntryDAO
+    extra = 0
+
+class PlayListAdmin(admin.ModelAdmin):
+    inlines = [PlayListEntryInline]
+
+# these 2 classes allow editing of reviews on the play admin page.
+# see https://docs.djangoproject.com/en/1.8/intro/tutorial02/#adding-related-objects
+class ReviewInline(admin.TabularInline):
+    model = ReviewDAO
+    extra = 0
+
+class PlayAdmin(admin.ModelAdmin):
+    inlines = [ReviewInline]
```

Comment:
Hey @B3nD3lan3y, take a look at the improved admin site to see if it might be adequate for approving reviews (if a check box was added for that)